jira-slack-server-integration/jira-slack-server-integration-plugin/src/main/resources/atlassian-plugin.xml

```diff
@@ -744,8 +743,7 @@
         <resource name="oauth2/view-slack-oauth-sessions.less.css" type="download" location="oauth2/view-slack-oauth-sessions.less"/>
     </web-resource>
 
-    <servlet name="View OAuth Slack Servlet" key="slack-view-oauth-servlet"
-             class="com.atlassian.plugins.slack.oauth2.ViewSlackOauthSessionsServlet">
+    <servlet name="View OAuth Slack Servlet" key="slack-view-oauth-servlet" class="com.atlassian.plugins.slack.oauth2.ViewSlackOauthSessionsServlet">
```

Comment:
Please revert all changes in this file. I understand that they are just formatting changes, but let's leave the file intact, since it is totally unrelated to the main PR goal.